corehq/apps/hqadmin/corrupt_couch.py

```diff
@@ -29,15 +27,14 @@
 from corehq.motech.repeaters.models import Repeater
 from corehq.util.couch_helpers import NoSkipArgsProvider
 from corehq.util.pagination import ResumableFunctionIterator
+from toggle.models import Toggle
 
 log = logging.getLogger(__name__)
 COUCH_NODE_PORT = 15984
 DOC_TYPES_BY_NAME = {
-    "forms": {
-        "type": XFormInstance,
-        "date_range": True,
-        "use_domain": True,
-        "doc_types": [
+    "main": {
+        "type": Toggle,
```

Comment:
I'm confused where `Toggle` came from. Did you need to pick something arbitrary as a placeholder, or is there something deeper?